sirepo/package_data/static/en/css/landing.css.map

```diff
@@ -1,7 +1,7 @@
+"sources": ["file:///Users/connor/Sites/cm-sirepo/sirepo/package_data/static/en/css/main.scss","file:///Users/connor/Sites/cm-sirepo/sirepo/package_data/static/en/css/variables.scss","file:///Users/connor/Sites/cm-sirepo/sirepo/package_data/static/en/css/responsive.scss"],
```

Comment:
It would be good to not have your home directory hardwired into this file. Is there a way to generate it with relative paths?

Reply:
I certainly agree! The task runner wasn't in the repo from the original build. I must have copied a bit too much when I was setting it up. I cleaned up the sass files and grunt process, so those paths should be relative now.